app/helpers/application_helper.rb

```diff
@@ -48,4 +48,8 @@ def workflow_grid_view?
   def profile_view?
     current_page?(search_profile_path)
   end
+
+  def avalaible_ocr_languages
```

Comment:
Can you put this in a component? That way it isn't in the global view scope.  Also, typo on the method name.

Reply:
It was used in a couple spots (bulk job view and modal), so I made a new component, moved the method there, fixed the typo, and used the component in both spots instead of the partial.